worlds/sc2/Client.py

```diff
@@ -21,12 +21,13 @@
 from CommonClient import CommonContext, server_loop, ClientCommandProcessor, gui_enabled, get_base_parser
 from Utils import init_logging, is_windows, async_start
 from worlds.sc2 import ItemNames
-from worlds.sc2.Options import (MissionOrder, KerriganPrimalStatus, kerrigan_unit_available, KerriganPresence, GameSpeed,
-    GenericUpgradeItems, GenericUpgradeResearch, ColorChoice, GenericUpgradeMissions,
-    LocationInclusion, ExtraLocations, MasteryLocations, ChallengeLocations, VanillaLocations,
-    DisableForcedCamera, SkipCutscenes, GrantStoryTech, TakeOverAIAllies, RequiredTactics, SpearOfAdunPresence,
-    SpearOfAdunPresentInNoBuild, SpearOfAdunAutonomouslyCastAbilityPresence, SpearOfAdunAutonomouslyCastPresentInNoBuild
-)
+from worlds.sc2.Options import MissionOrder, KerriganPrimalStatus, kerrigan_unit_available, KerriganPresence, \
+    GameSpeed, GenericUpgradeItems, GenericUpgradeResearch, ColorChoice, GenericUpgradeMissions, \
+    LocationInclusion, ExtraLocations, MasteryLocations, ChallengeLocations, VanillaLocations, \
+    DisableForcedCamera, SkipCutscenes, GrantStoryTech, TakeOverAIAllies, RequiredTactics, SpearOfAdunPresence, \
+    SpearOfAdunPresentInNoBuild, SpearOfAdunAutonomouslyCastAbilityPresence, \
+    SpearOfAdunAutonomouslyCastPresentInNoBuild
```

Comment:
Now where'd the parens go? Just do:
```python
from worlds.sc2.Options import (
    MissionOrder, KerriganPrimalStatus, kerrigan_unit_available, KerriganPresence,
    # ...
)
```
And you never need an EOL backslash again. Is this PyCharm doing autoformatting?

Reply:
yeah, the autoformating when auto-add import does that formatting, when it's backshalsh'd, the problem doesn't exist